src/Sylius/Behat/Page/ElementNotFoundException.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Behat\Page;
+
+/**
+ * @author Kamil Kokot <kamil.kokot@lakion.com>
+ */
+final class ElementNotFoundException extends \Exception
```

Comment:
:+1:, but since it's not a critical feature, I think it can be merged as is and fixed just after merge, I don't want to wait with this PR for another build to complete as it touches the part that is actively developed right now.